conda-forge.yml

```diff
@@ -1,4 +1,4 @@
 conda_forge_output_validation: true
 provider: {linux_aarch64: false, linux_ppc64le: azure}
 choco:
-- cuda --version=11.1.1
+- cuda --version=11.2.2.46133
```

Comment:
Does this need to be patched somewhere else? @jakirkham

Reply:
![Screen Shot 2021-03-16 at 14 25 58](https://user-images.githubusercontent.com/1915404/111360771-8b301300-8663-11eb-9d73-eb1ced8e89ad.png)
Looks like a recent change for 11.2.1+

Reply:
Yup, I just vaguely remember that this file is autogenerated from somewhere else and that a re-render would blow away these changes. I could be mistaken though.

Reply:
Yeah I need to rerender once more to get the change propagated and get the windows builds working

Reply:
I think it is just a re-render, but let's check with @jaimergp 🙂

Reply:
I think we need that to provide the drivers in case we want to run some tests, but without getting into a catch 22 situation if we use `conda-forge-ci-setup` provided installers. If that's correct, any sufficiently recent one should do, right? Maybe @scopatz better recalls why `choco` was needed.

Reply:
So what would be the next step here? Merge this and open a PR to `conda-forge-ci-setup`? Reverse order? Something else?

Reply:
^ @jaimergp?